vector/src/main/java/im/vector/app/features/widgets/WidgetFragment.kt

```diff
@@ -227,6 +227,9 @@ class WidgetFragment :
                 views.widgetProgressBar.isVisible = true
             }
             is Success -> {
+                if (views.widgetWebView.url == null) {
+                    loadFormattedUrl(formattedUrl())
```

Comment:
Is there a risk that we load the webview twice (as we're already triggering `WebView.loadUrl()` [here](https://github.com/vector-im/element-android/pull/8290/files#diff-7bab8af36be71f0cd9ab4b960fad2189e9ffbb53710a66794c9dd41452159c72R99))?

Do we know why the other `loadUrl()` is not working?

Reply:
> Is there a risk that we load the webview twice

I don't think so because the webview url should be not null after a first url loading.

> Do we know why the other loadUrl() is not working?

Yes it's a regression after [this change](https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/core/utils/SharedEvent.kt#L41) from [this PR](https://github.com/vector-im/element-android/pull/7917)
The view event is posted before the webview is ready to receive it.